api/package.json

```diff
@@ -23,7 +23,8 @@
         "reindex": "ts-node prisma/reindex.ts",
         "format:check": "npx prettier --check src/",
```

Comment:
these can go? prettier has been removed?

Reply:
we've decided to keep it and only do formatting using it, eslint just does linting